### ISSUE TITLE
accessibility issue for ui5-checkbox

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#1664]
**Describe the bug**
ui5-checkbox has internally managed div in shadow dom with role checkbox  e.g.

<div role="checkbox" class="ui5-checkbox-root ui5-checkbox--hoverable" aria-checked="false" tabindex="-1">

accessibility test fails since this div does not have the aria-label for the screen readers. 

**Expected behavior**
if text attribute is provided, can that be added as aria-label for the internal div e.g.
<ui5-checkbox text="Approved" checked></ui5-checkbox>
inside shadowdom of ui5-checkbox, assign for the <div role="checkbox"... aria-label={toggle Approved}>

**Screenshots**
![Screen Shot 2020-05-21 at 12 37 28 AM](https://user-images.githubusercontent.com/65705637/82537867-9ed02300-9aff-11ea-93c9-bd85bf157af4.png)

**Comment by ilhan007** [comment]
Hello @gurkirpalgill 

Currently, in the root tag of the shadow root of the CheckBox we have aria-labelledby:

```html
<div
	role="checkbox"
	aria-checked="{{checked}}"
	aria-labelledby="{{ariaLabelledBy}}"
         ...
>
```

The aria-labelledby is present whenever the CheckBox has text. If there is no text set, there is no aria-labelledby.  Or, it must be aria-label, can you set a text to get the aria-labelledby and re-check with the tool?

Best,
ilhan

**Comment by gurkirpalgill** [comment]
I am not directly invoking <ui5-checkbox> I am instantiating <ui5-li> inside:
<ui5-list id="list" mode="MultiSelect"> ...</ui5-list>  ui5-li internally instantiates ui5-checkbox. Hence I do not have access to ui5-checkbox in a straightforward say.  Here is the screenshot example from:
https://sap.github.io/ui5-webcomponents/playground/components/List/

As you can see in the screenshot, it has the same accessibility issue.
![Screen Shot 2020-05-22 at 5 47 15 PM](https://user-images.githubusercontent.com/65705637/82718086-1f546800-9c55-11ea-87d4-5120cadcd69a.png)
